src/librbd/cache/pwl/ssd/Request.cc

```diff
@@ -34,10 +34,10 @@ template <typename T>
 std::ostream &operator<<(std::ostream &os,
                          const C_CompAndWriteRequest<T> &req) {
   os << (C_WriteRequest<T>&)req
-     << "cmp_bl=" << req.cmp_bl << ", "
-     << "read_bl=" << req.read_bl << ", "
-     << "compare_succeeded=" << req.compare_succeeded << ", "
-     << "mismatch_offset=" << req.mismatch_offset;
+     << " cmp_bl=" << req.cmp_bl << ","
+     << " read_bl=" << req.read_bl << ","
```

Comment:
In cases like this, that is within a single logging statement, I would merge `","` into the head space.  Less `operator<<` calls, more performance.
```suggestion
     << " cmp_bl=" << req.cmp_bl <<
     << ", read_bl=" << req.read_bl << ","
```

Reply:
According to your suggestion, I merged most of the `“<<”` calls that can be merged in PWL in new commit. Please check is it OK?